addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -234,10 +234,10 @@ const Link = Widget.extend({
      * @private
      */
     _correctLink: function (url) {
-        if (url.indexOf('mailto:') === 0 || url.indexOf('tel:') === 0) {
-            url = url.replace(/^tel:([0-9]+)$/, 'tel://$1');
-        } else if (url.indexOf('@') !== -1 && url.indexOf(':') === -1) {
+        if (url.indexOf('@') >= 0 && url.indexOf('mailto:') === -1 && !url.match(/^http[s]?/i)) {
             url = 'mailto:' + url;
+        } else if (/^[^a-zA-Z]*[0-9]+[^a-z]*$/.test(url)) {
```

Comment:
Why `[^a-z]` and not `[^a-zA-Z]` ?
Is this for the american phone number with letters?
If thats the case:
1. I give you 10 bonus point for taking that into consideration
2. I remove 1 bonus point for not adding a comment about it (no one will ever know).

Note that it will recognize `01NET.COM` as a phone but I guess we can live with that

Reply:
No extra point then :D

Reply:
Nope... that was just a mistake :grimacing:

Reply:
Where can I spend RDE points? :open_mouth: